addons/rose/tests/index.html

```diff
@@ -7,7 +7,6 @@
 <html>
   <head>
     <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
None blocking comment: was this done by the blueprints?

Reply:
yup, that is correct!